src/recommendations/data/service.js

```diff
@@ -0,0 +1,22 @@
+import { camelCaseObject } from '@edx/frontend-platform';
+import algoliasearch from 'algoliasearch/lite';
+
+const INDEX_NAME = process.env.ALGOLIA_AUTHN_RECOMMENDATIONS_INDEX;
+
+const getPersonalizedRecommendations = async (level) => {
+  const educationLevel = level ? `level:${level}` : '';
+
+  const client = algoliasearch(process.env.ALGOLIA_APP_ID, process.env.ALGOLIA_SEARCH_KEY);
+  const index = client.initIndex(INDEX_NAME);
+
+  const { hits } = await index.search('', {
+    aroundLatLngViaIP: true,
+    facetFilters: [
+      'product:Course', 'availability:Available now', educationLevel,
+    ],
+  });
+
+  return camelCaseObject(hits);
+};
```

Comment:
If education level is not present we should not send it.  Also lets move the filter into a variable so that its easier to manipulate
```suggestion
const getPersonalizedRecommendations = async (educationLevel) => {
  const facetFilters = ['product:Course', 'availability:Available now'];
  
  if (educationLevel) {
    facetFilters.push(`level:${educationLevel}`);
  }

  const client = algoliasearch(process.env.ALGOLIA_APP_ID, process.env.ALGOLIA_SEARCH_KEY);
  const index = client.initIndex(INDEX_NAME);

  const { hits } = await index.search('', {
    aroundLatLngViaIP: true,
    facetFilters,
  });

  return camelCaseObject(hits);
};
```